### PR TITLE
Set up google analytics

### PR DESCRIPTION
**Pull request by jenlu33**
I followed a tutorial and set up Google analytics here. I added the tracking ID into the credentials file, but I had to generate a master key before doing that. Let me know if I need to share the key or if I should have set this up in a different way

### CODE REVIEW

File: app/views/layouts/_google_analytics.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% if Rails.env.production? %>
+<script async src="https://www.googletagmanager.com/gtag/js?id=<%= Rails.application.credentials.dig(:google_analytics) %>"></script>
```
Comment (littleforest) [inline app/views/layouts/_google_analytics.erb:2]: Picky: indent 2 spaces

**Review by littleforest — COMMENTED** [review]
@jenlu33 thanks for working on this! You actually don't need to keep the Google Tag ID in secrets, since it will end up being publicly viewable in the JavaScript anyway. So you can just copy and paste the snippet as-is.

Do you have the link for the tutorial that you followed? I'm not sure why we can't just put the snippet in the `head` tag [per the Google Analytics docs](https://support.google.com/analytics/answer/9304153?hl=en#zippy=%2Cadd-the-google-tag-directly-to-your-web-pages).

**Comment by jenlu33** [comment]
@littleforest Here's the tutorial I used [https://michaelsoolee.com/google-analytics-rails-5/](https://michaelsoolee.com/google-analytics-rails-5/)

**Comment by littleforest** [comment]
Oh yes, turbolinks gets me every time. Let's see, we don't have asset pipeline set up; we are using[ import maps](https://github.com/rails/importmap-rails) for JS. Can you move the js file to: `app/javascript/google_analytics.js` (you can remove the `.erb` extension and just hard-code the Google tag), and then add:

1.  `import "google_analytics"` to `app/javascript/application.js`
2.  `pin "google_analytics"` to `config/importmap.rb`

And we can see if that works? I will be around to pair at 2:30 EST if needed.

**Comment by littleforest** [comment]
@jenlu33 also we are using just Turbo now (replaces Turbolinks), so I think we probably need to do just `turbo:load` per [this comment](https://github.com/turbolinks/turbolinks/issues/73#issuecomment-812484452).

**Comment by jenlu33** [comment]
@littleforest  Just made all the changes! Is there a way for me to test if it works?

File: config/credentials.yml.enc [hunk 2]
```diff
@@ -1 +1 @@
-Zl4ZmIChG4IqsN8yFJ+Y714y+ym1Mr7PCRPy7xbu9HOdMzX1/c6VxbYTJUhtqTbLHrp2LHCkZTfz15t79lDiwuBp00VtoWB29kC3UtSYsYKebSZsxPBfL/YfG3JrtNUgdtXMzoTbatW5tvVglKDSdQpkihdH1GIaq1pnzJaeW85Jgds3i2npYW3eeeuQI4/Ppp9U3nHoW+0daxMZxfJg6OdFZ3Xa5RDDNH75zgpObKjEcXZF9VZYqyScMDwH2N3OOe4YtIRrLDRaw/tnAauFp7wM5QjpWBegsKh3eM31hV8M1arHnHlyO+EWN2NRTrmKsUKWmewrOFu9w+NiNmNEFDKzb6/NLbPDb1+3FSxIf6515SxhgktkodL5qZoNffYXm6R3uQLfGkzlM4dFs/u+c4O4ryXjC7ols+6R--XYxLKDdGWAZhwCSX--Jp7XoPCoaaLe1PtnUiPWHA==
\ No newline at end of file
+PYhAQqx1QPMgNIUD8yy2Ck9Gm2KXEI6j4T4h8D6R5yRNCelUfFZ/mTdv1VXdOT+1j8M+VXy+Adw7Ug1/aJ78VGa5Ic0GkJL+J+a8fyoKNXpRlE+RbQ66kDPCDtlpYU6f90GHfEYFqimUySAHrHIPfs5HM0qiSU5K4M5Rt7C7h0NpCjOzCpu7Cu3R6QGucLOITKotBNEcRo8DzoKF8bRj5Eg+lx3ZxSczS5ybq6GSPZxbQ/dRguJ84K/tY0USYU6DE4eIa4qqpFdlMqdQMWD5gj4YgtCeIgQrFSVnpbZ2bJlDv47xl5mKU4/IunomvohRyqSFki2GOFF9bEORkOp+SE6CVkOa8IXa6iHlyn+Zg98b8HMkj/0Da4KC8LYDxYSN1rI2aAnxpgTRK33+/lgogLYUic3Vc7ZPRaOSSFxK3SQERyR8bCEhyKVL5w6TrJIUa5f2--wnl1nth/REa83tR5--N+C+RU/Rcnqf09O/8oKETA==
```
Comment (littleforest) [inline config/credentials.yml.enc:1]: @jenlu33 can you revert this change since we didn't add anything to the credentials?

```
git checkout main -- config/credentials.yml.enc
```

**Comment by littleforest** [comment]
@jenlu33 I'm going to push to Heroku from the local branch and see if anything shows up. You should have access to the Google Analytics page I believe. Try going to `analytics.google.com`.

**Review by littleforest — APPROVED** [review]
It seems to be working!

Screen Shot 2023-01-03 at 11 56 02 AM

**Comment by jenlu33** [comment]
> It seems to be working!
> 
> Screen Shot 2023-01-03 at 11 56 02 AM

Great! It doesn't seem like I have access to Google Analytics though? I'm just prompted to create an account to start measuring when I log in with my Workhands account
